resources/migrations/001_update_migrations.yaml

```diff
@@ -8249,10 +8249,11 @@ databaseChangeLog:
               - column:
                   name: device_id
       rollback:
-        - dropIndex:
+        - dropIndex
```

Comment:
Please do not forget to change this back before merging

Reply:
yes it was only for testing the auto request workflow. 

the BE test will fail with this change anyway because it's invalid yaml